vagrant/k8s/Vagrantfile

```diff
@@ -85,9 +85,7 @@ echo "export no_proxy='k8master,192.168.2.10,192.168.2.11,192.168.2.12,192.168.2
 source /etc/profile.d/envvar.sh
 # Change ownership for gopath folder
 sudo chown -R vagrant #{gopath_folder}
-sudo yum install -y net-tools
-sudo yum install -y go
-sudo yum install -y git
+sudo yum install -y net-tools git
```

Comment:
Go isn't needed because we already have Go 1.7.6 in the box. This avoids the need to fetch a package which isn't going to be used.

Installing the packages at the same time is a good idea.